source/_integrations/snmp.markdown

```diff
@@ -438,4 +442,17 @@ switch:
     payload_on: 15400
     payload_off: 3000
     vartype: Gauge32
+ 
+  - platform: snmp
```

Comment:
String values need to be enquoted. See our style guide for more information: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide>